cpp/CMakeLists.txt

```diff
@@ -55,6 +55,29 @@ rapids_find_package(
   INSTALL_EXPORT_SET kvikio-exports
 )
 
+# If AWSSDK isn't found, the Cython module remote_handle.pyx isn't built and C++ users shouldn't
+# include <kvikio/remote_handle.hpp>
+rapids_find_package(
```

Comment:
*question*: Perhaps not for this prototype, but if it is easy, perhaps yes.

Should building S3 support be a user-configurable option. That is, even if we can find the package, should we have an option (say `-DKVIKIO_HAVE_S3_SUPPORT=1`) to cmake so that the user can compile without this dependency if they are only doing local I/O?

Reply:
Good point, done